src/ape_ethereum/ecosystem.py

```diff
@@ -935,14 +935,14 @@ def get_abi(_topic: HexStr) -> Optional[LogInputABICollection]:
                     converted_arguments[key] = value
 
             yield ContractLog(
-                block_hash=log["blockHash"],
-                block_number=log["blockNumber"],
+                block_hash=log.get("blockHash", log.get("block_hash")),
+                block_number=log.get("blockNumber", log.get("block_number")),
```

Comment:
To be a little lazier
```suggestion
                block_hash=log.get("blockHash") or log.get("block_hash"),
                block_number=log.get("blockNumber") or log.get("block_number"),
```

Reply:
i like some of this but maybe not for the ints, as it would do `0 or None` and then be `None` instead of `0`.

Reply:
or could do  a final `or 0`

Reply:
and transaction index can be None so, also will keep that in mind

Reply:
what scenario can it be none? when it's pending block?

Reply:
oh, I don't know, I was only looking at the type

Reply:
yes, pending block, according to the doc str: https://github.com/ApeWorX/ape/blob/main/src/ape/types/__init__.py#L279